leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/serialization/EndpointContextSerDes.java

```diff
@@ -78,7 +79,11 @@ public static JsonObject serialize(EndpointContext context) {
             JsonObject attContext = Json.object();
             for (String key : attributes.keySet()) {
                 // write all values as string
-                attContext.set(key, attributes.get(key).toString());
+                Object value = attributes.get(key);
+                if (value instanceof Bytes) {
```

Comment:
I was right for bad reason because I thought that byte array was `byte[]` and not `Bytes` :sweat_smile:

Reply:
:-) I found so a bug in `AddressEndpointContext.getString(String key)`. `Bytes.toString()` is for logging ...

Reply:
About  `AddressEndpointContext.getString(String key)`, 

I read those code and I saw that the behavior is different than the getNumber() or getBytes()

I mean getNumber and getBytes will failed if the value is not of the expected type and getString will never failed (as toString() is called instead of a cast in String). 

Is it expected ?

Reply:
The intention was to provide a "shortcut for migration". Anyway, it's till open, if the Attributes are changing "again" to a similar API as Configuration. I'm still working the Configuration and DTLS.